tests/integration/techRecords.intTest.ts

```diff
@@ -21,8 +22,10 @@ describe("techRecords", () => {
         const responseObj = Object.assign({}, dbObj);
 
         // Adding primary and secondary VRMs in the same array
-        const vrms: any = [{ isPrimary: true }];
-        if (responseObj.primaryVrm) { vrms[0].vrm = responseObj.primaryVrm; }
+        const vrms: any = [{isPrimary: true}];
```

Comment:
is this needed to fix a data issue? Is it better fixed in the data?

Reply:
No, from what i've seen this is used to transform the returned object and it just changes it's structure to match the returned object's structure from the getTechRecordsList function. That's all. This wasn't written by me btw.

Reply:
Yeah, I saw you just reshaped it. I strongly suspect that this is a kludge used to get around the fact that our data is wrong. I'll worry about it later.